baiji/config.py

```diff
@@ -1,7 +1,7 @@
 import os
 from baiji.exceptions import AWSCredentialsMissing
 
-class Credentials(object):
+class Settings(object):
```

Comment:
Yeah they share the same loading path (in fallback order: aws cli well known path -> environment variable)